src/GlView.h

```diff
@@ -30,14 +30,15 @@
 #include <QTime>
 #include <QBasicTimer>
 #include <QGLWidget>
```

Comment:
Is `QGLWidget` still required at all? (If not it would be nice to remove the include.)